packages/itmat-job-executor/test/unit_tests/csvcurator.test.ts

```diff
@@ -1,5 +1,7 @@
 import { processDataRow, processHeader, CSVCurator } from '../../src/curation/CSVCurator';
 import fs from 'fs';
+import { stub } from './_stubHelper';
+import { IJobEntry } from 'itmat-commons/dist/models/job';
```

Comment:
IJobEntry import from 'itmat-commons' directly instead of in /dist?

Reply:
Imports from `itmat-*/dist/` are used in about a hundred files. As such we cannot get `IJobEntry` directly from `itmat-commons` because it is not getting expose by the package. I'd rather do a separate PR and have all import from `itmat-*/dist/` modified arccordingly. Would that work for you ?

Reply:
isn't it exposed in this way?
import { Models } from 'itmat-commons';
const a: Models.JobModels.IJobEntry<any> = //////

Let's do another PR for this then